### PR TITLE
Restrict bug logging for unmapped resources

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json [hunk 1]
```diff
@@ -369,7 +369,8 @@
     "ServiceTreeIdType": "Service",
     "UseAzureStorageAccount": false,
     "LogBugsForInactiveResources": true,
-    "CustomControlList": []
+    "CustomControlList": [],
+    "LogBugForUnmappedResource": true
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:373]: LogBugsForUnmappedResource

File: src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1 [hunk 2]
```diff
@@ -15,6 +15,7 @@ class AutoBugLog {
     hidden [bool] $UseAzureStorageAccount = $false;
     hidden [BugLogHelper] $BugLogHelperObj;
     hidden [string] $ScanSource;
+    hidden [bool] $LogBugForUnmappedResource = $true;
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1:18]: LogBugsForUnmappedResource

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json (hunk 1 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json]: Renamed

File: src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1 (hunk 2 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1]: Renamed

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**